commons/src/main/java/org/sonarsource/analyzer/commons/RuleMetadataLoader.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Sorry, I missed this. This is not expected to be shipped, please drop the annotation. It is not part of the annotations we support in our code.